core/src/com/unciv/logic/map/TileInfo.kt

```diff
@@ -352,6 +352,8 @@ open class TileInfo {
                     && improvement.name != "Remove Road" && improvement.name != "Remove Railroad"
                     && getTileImprovement().let { it != null && it.hasUnique("Irremovable") } -> false
 
+            // Decide cancelImprovementOrder earlier, otherwise next check breaks it
+            improvement.name == Constants.cancelImprovementOrder -> (this.improvementInProgress != null)
```

Comment:
Unneeded parentheses :)

Reply:
This is just a - what's it called - [idée fixe](https://en.wikipedia.org/wiki/Id%C3%A9e_fixe_(psychology)) of mine that enclosing boolean expressions that are immediately passed on (like in a return statement) like this _improves readability_ - bool expressions often contain the <> characters, the when condition -> block delimiter, too - so the eye is guided a little to quicker recognize what associates with what...